packages/idealo.design/src/server/db.js

```diff
@@ -55,11 +55,11 @@ export async function storeSinglePost({
 
 export async function updateSinglePost(blog) {
   const updatedPost = await sql`
-    update blogposts set ${
-      sql(blog, 'text', 'title' )
-  } where 
-    id = ${ blog.id }
-`     
+  update blogposts set ${
+      sql(blog, 'text', 'title', 'categoryDisplayValue', 'categorySlug')
+  } where
+  id = ${ blog.id }es
```

Comment:
I think the "es" got added here by accident @LuciaFeddersen. It produces an error.

Reply:
yes sorry i will change it

Reply:
thank you for changing it